app/server/server.go

```diff
@@ -46,7 +47,7 @@ func (s *Rest) Run(ctx context.Context) error {
 		Addr:              s.Listen,
 		Handler:           s.router(),
 		ReadHeaderTimeout: time.Second,
-		WriteTimeout:      30 * time.Second,
+		WriteTimeout:      s.Timeout + 10*time.Second, // Give enough time to finish the task and respond
```

Comment:
Looks like it takes the user-defined duration and multiplies it by 10 seconds? I would expect the timeout to match what the user actually set in the CLI.

Another thing, for backward compatibility, the default should be the same as before, i.e., 30s.

Reply:
@umputun As far as I remember there is 2 timeouts in different places, and when we tested that with @paskal on hardcoded 30 seconds we've got ambiguous errors. Anyway I am ok to revert this change if you sure that we need that.

Reply:
Ok, I kind of remember this story too.